### DISCUSSION TITLE
Does dynamic filters work for spring?

### DISCUSSION THREAD

**Discussion by rockstar123-bit** [graphql-java/graphql-java#2776] [Q&A]
Hi Team,

     Is there a way to add dynamic filters in graphql with spring? i'm trying to find something like below :
 
```
query{
findStatus(status: active or inactive)
}
```
 could someone guide me on doing this?

 thanks in advance.

**Comment by dugenkui03** [comment]
@rockstar123-bit Look like same thing https://github.com/graphql-java/graphql-java/discussions/2772
